utilities/shared_disk.py

```
@@ -0,0 +1,144 @@
+
+from utilities.post_migration import get_ssh_credentials_from_provider_config
+from utilities.ssh_utils import SSHConnectionManager
+
+if TYPE_CHECKING:
+    from kubernetes.dynamic import DynamicClient
+
+LOGGER = get_logger(name=__name__)
+
+
+def _run_cmd_on_vm(
+    ssh_conn: Any,
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `ssh_conn`

(ANN401)